config/sync/block.block.da_vinci_footer.yml

```diff
@@ -1,6 +1,6 @@
 uuid: ac927b70-8fde-419d-8550-8072b8e76731
 langcode: en
-status: false
+status: true
```

Comment:
For what is this block used? For placing the menu link to the contact form? If it is the case. I'd like to avoid placing any menu block at this moment because we don't know where we'll place the menu link to the contact.